application/core/calendar/calendarInvite.go

```diff
@@ -110,15 +114,43 @@ func SendInvite(matches []matcher.Match) (string, error) {
 				}
 			case mailErrors := <-errorschannel:
 				fmt.Println("received", mailErrors)
+
 				if len(mailErrors) > 0 {
 					database.UpdateJobErrors(jobId, mailErrors)
 				}
+				saveMsg, err := saveMatchingInfo(len(matches), len(mailErrors))
+
+				if err != nil {
+					database.UpdateJobErrors(jobId, []string{saveMsg})
```

Comment:
be careful this is not good you are erasing all errors put at line 121.
so if you update you should append first this error to the mailErrors and put all

Reply:
so here, what I append on MailErrors list: saveMsg or error? I gonna to put error. it's right?